crates/jetstreamer-source/Cargo.toml

```
@@ -0,0 +1,44 @@
+[package]
+name = "yellowstone-vixen-jetstream-source"
+version = "0.3.1"
```

```suggestion
version.workspace = true
```

Going to start publishing crates all under the same version for simplicity.